internal/pkg/mapper/mapper.go

```diff
@@ -45,10 +45,39 @@ type OnRemoteChangeListener interface {
 	OnRemoteChange(changes *model.RemoteChanges) error
 }
 
+type Mappers []interface{}
+
+func (m Mappers) ForEach(stopOnFailure bool, callback func(mapper interface{}) error) error {
```

Comment:
Pridane metody `ForEach` a `ForEachReverse` ... go nema ziadnu nativnu funkciu, ktora by to riesila.
Je mozne zadat aj `stopOnFailure`.